spec/rspec/core/reporter_spec.rb

```diff
@@ -36,6 +35,12 @@ module RSpec::Core
         reporter.finish
       end
 
+      it "will allow profiler to be used without being properly setup" do
```

Comment:
Thanks, this wording is a lot more clear.

s/will allow/allows/ (shorter and more inline with the phrasing we use elsewhere).

Reply:
I've reworded.